README.md

```diff
@@ -1,2 +1,10 @@
 # react-native-oauth2
 OAuth2 Package for RN
+
+## Usage
+```javascript
+import Oauth2 from 'react-native-oauth2'
+
+Oauth.authenticate()
```

Comment:
Promisee should be returned from both as these are async operations.

Reply:
removed.